android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OnMethodCallMapper.kt

```diff
@@ -93,18 +96,24 @@ class OnMethodCallMapper(private var context: Context, private val oneginiMethod
 
             Constants.METHOD_VALIDATE_PIN_WITH_POLICY -> validatePinWithPolicy(call.argument<String>("pin")?.toCharArray(), result, client)
 
-            else -> result.error(OneginiWrapperErrors.METHOD_TO_CALL_NOT_FOUND.code, OneginiWrapperErrors.METHOD_TO_CALL_NOT_FOUND.message, null)
+            else -> SdkError(
+                wrapperError = OneWelcomeWrapperErrors.METHOD_TO_CALL_NOT_FOUND
+            ).flutterError(result)
         }
     }
 
     // "https://login-mobile.test.onegini.com/personal/dashboard"
     fun getAppToWebSingleSignOn(url: String?, result: MethodChannel.Result, oneginiClient: OneginiClient) {
         if (url == null) {
```

Comment:
this is intended, since the .flutterError(result) call sends the resulting error over the connection bridge to the flutter side of things and there it gets further handled, the return later is to finish the things on kotlin side :) 

side note taken!